pkg/ipmi/aes_128_cbc_test.go

```diff
@@ -47,8 +47,30 @@ func TestAES128CBCDecodeFromBytes(t *testing.T) {
 				0x3a, 0x3c, 0x9b, 0x30, 0x81,
 			},
 		},
-		// 16 bytes data (must flow onto second block for padding length, max
-		// padding)
+		// 15 bytes of data encrypted with 16 bytes of padding.
+		// This is not how AES128CBC.SerializeTo would encrypt this message, but
+		// other implementations of AES in CBC mode may work this way (e.g.
+		// OpenSSL).
+		{
```

Comment:
This test fails with `unexpected error: invalid number of pad bytes: 16` if we remove the `pkg/ipmi/aes_128_cbc.go` change.